deep_qa/data/datasets/language_modelling/language_modelling_dataset.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+from overrides import overrides
+
+from ..dataset import TextDataset, count_labels
+from ...instances import TextInstance
+from ...instances.language_modeling import SentenceInstance
+from ....common.params import Params
+
+
+class LanguageModellingDataset(TextDataset):
+
+    def __init__(self, instances: List[TextInstance], params: Params=None):
+        self.sequence_length = params.pop("sequence_length")
+        super(LanguageModellingDataset, self).__init__(instances)
+
+    @staticmethod
+    @overrides
+    def read_from_file(filename: str, instance_class, params: Params=None):
+
+        sequence_length = params.get("sequence_length", 20)
+        with open(filename, "r") as text_file:
+            text = text_file.readlines()
+            text = " ".join([x.replace("\n", " ").strip() for x in text]).split(" ")
+
+        instances = []
+        for index in range(0, len(text) - sequence_length, sequence_length):
+            word_sequence = " ".join(text[index: index + sequence_length])
```

Comment:
Splitting on spaces like this seems a little problematic to me, actually.  You really should be doing this with a tokenizer.  But we can worry about that later, if/when we actually use this for language modeling.  Can you at least leave a note here that we might want to revisit how this is done?